models/staging/stg_ant_ingest.sql

```diff
@@ -0,0 +1,50 @@
+{{ 
+    config(
+        materialized='incremental',
+        unique_key='ingest_timestamp',
+        incremental_strategy = 'delete+insert',
+        tags=['core','ingest'],
+        cluster_by=['ingest_timestamp']
+        )
+}}
+
+with
+old_source_table as (
+
+    select
+        ingest_timestamp::timestamp as ingest_timestamp,
+        try_parse_json(ingest_data) as parsed_data
+    from {{ source("ingest","src_old_ant_ingest") }}
+    where     
+        {% if is_incremental() %}
+            ingest_timestamp::timestamp > (select max(ingest_timestamp) from {{ this }})
+        {%- else -%}
+            true
+        {% endif %}
```

Comment:
isn't it the same as the macro for incremental build?

Reply:
actually how come you have old and new ingest data?

Reply:
[Macro] There's a cast on ingest_timestamp. (Ingest Table is setup to be varchar[16mb])
Old Table = copied over the old data from harmony,
new Table the current ingest table. There's some overlap in the timing thus why its setup this way...